### PR TITLE
Add review process documentation

### PR DESCRIPTION
**Pull request by Purg**

### CODE REVIEW

File: docs/review_process.rst [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+^^^^^^
+Indicates that the PR is resolved or discarded.
+
+
+Continuous Integration
+======================
+The following checks are included in the automated portion of the review
+process.
+These are run as part of the CI/CD pipeline driven by GitHub actions.
+The success or failure of each may be see in-line in a submitted PR in the
```
Comment (brianhhu) [inline docs/review_process.rst:51]: "may be see" -> "may be seen"

File: docs/review_process.rst [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+ghostflow
+---------
+Runs basic checks on the commits submitted in a PR.
+Should ghostflow find any issues with the build, the diagnostics are written
+out prompting the submitter to correct the reported issues.
+If there are no issues, or just warnings, ghostflow simply reports a successful
+build.
+The branch should usually pass this check before it can be merged.
+In the rare case that a PR is not subject to a change note, then failure of
+this check specifically in regard to that lower-level check, may be ignore by
```
Comment (brianhhu) [inline docs/review_process.rst:63]: "may be ignore" -> "may be ignored"

File: docs/review_process.rst [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+or added in the PR.
+The configuration for this may be found in the ``codecov.yml`` file in the
+repository root.
+
+Passage of these checks is strictly required.
+
+ReadTheDocs Documentation Build
+-------------------------------
+This check ensures that the documentation portion of the package is buildable
+by the current host ReadTheDocs.org.
```
Comment (brianhhu) [inline docs/review_process.rst:131]: What happens if the ReadTheDocs build fails, i.e. is this check strictly required (keeping in line with the other tests)?

**Review by brianhhu — COMMENTED** [review]
LGTM, do we want to add the review_process.rst file to the index, so that it shows up in ReadtheDocs?

File: docs/review_process.rst [hunk 4]
```diff
@@ -0,0 +1,163 @@
+SMQTK Review Process
```
[the remark this answers is not in the document]
Comment (Purg) [reply docs/review_process.rst:1]: Oops! Let me replace that.

File: docs/review_process.rst (hunk 1 above)
Comment (Purg) [reply docs/review_process.rst:51]: :heavy_check_mark:

File: docs/review_process.rst (hunk 2 above)
Comment (Purg) [reply docs/review_process.rst:63]: :heavy_check_mark:

File: docs/review_process.rst (hunk 3 above)
Comment (Purg) [reply docs/review_process.rst:131]: Good catch, I've added a similar line as the other "required" checks.

**Comment by Purg** [comment]
> LGTM, do we want to add the review_process.rst file to the index, so that it shows up in ReadtheDocs?

Indeed! Let me add the reference.

**Review by brianhhu — APPROVED** [review]
LGTM

**Pull request merged**